frame/nfts/src/lib.rs

```diff
@@ -278,6 +279,18 @@ pub mod pallet {
 	pub(super) type CollectionMaxSupply<T: Config<I>, I: 'static = ()> =
 		StorageMap<_, Blake2_128Concat, T::CollectionId, u32, OptionQuery>;
 
+	#[pallet::storage]
+	/// Handles all the pending swaps.
+	pub(super) type PendingSwapOf<T: Config<I>, I: 'static = ()> = StorageDoubleMap<
```

Comment:
Is there any other operations we expect on PendingSwaps, other than the ones we have in the PR?
Just to make us think one more time on the storage definition.

Reply:
> Is there any other operations we expect on PendingSwaps, other than the ones we have in the PR? 

afaik no